### PR TITLE
ABZ-42995 add Coverity scan to pipeline

### PR DESCRIPTION
**Pull request by crdandoukho**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -23,8 +23,48 @@ variables:
...
+            CovPwd: "$(CovPwd)"
+            StreamName: "file-share-dotnet-client"
+            BuildCommand: "$(Build.Repository.LocalPath)/file-share-dotnet-client/FileShareAdminClient/FileShareAdminClient.csproj"
+            StripPath: $(Build.Repository.LocalPath)/file-share-dotnet-client
+            CoverityScanPath: $(Build.Repository.LocalPath)/coverityscan-buildtemplates
+
+
   - stage: PERFORM_DEPENDENCYCHECK_DOTNETBUILD_DOTNETTEST_AND_PUBLISH
     displayName: "Build (inc DependencyChecker, Dotnet Build , dotnet test and publish artifact )"
     jobs:
```
Comment (barrie-cooper) [inline azure-pipelines.yml:70]: ?
```suggestion
    dependsOn: []
    jobs:
```

Comment (crdandoukho) [reply azure-pipelines.yml:71]: Good spot, thanks. Have added missing `dependsOn`

Comment (barrie-cooper) [reply azure-pipelines.yml:71]: Better 😃

**Review by barrie-cooper — APPROVED** [review]
LGTM

**Pull request merged**